casemanagementv1/case_management_v1_examples_test.go

```diff
@@ -28,8 +28,6 @@ import (
 
 	"github.com/IBM/go-sdk-core/v5/core"
 	"github.com/IBM/platform-services-go-sdk/casemanagementv1"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
```

Comment:
These imports are necessary. Without them cannot run the tests (Describe, Expect, It, etc...)